modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/info/InstanceInfo.java

```diff
@@ -19,19 +19,48 @@
 
 package org.nuxeo.launcher.info;
 
+import static java.util.Arrays.asList;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.nuxeo.launcher.config.ConfigurationConstants.ENV_NUXEO_PROFILES;
+import static org.nuxeo.launcher.config.ConfigurationConstants.FILE_TEMPLATE_DISTRIBUTION_PROPS;
+import static org.nuxeo.launcher.config.ConfigurationGenerator.TEMPLATE_SEPARATOR;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
 
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlElementWrapper;
 import javax.xml.bind.annotation.XmlRootElement;
 
+import org.nuxeo.common.Environment;
+import org.nuxeo.common.codec.Crypto;
+import org.nuxeo.connect.update.LocalPackage;
+import org.nuxeo.launcher.config.ConfigurationHolder;
+
 @XmlAccessorType(XmlAccessType.NONE)
 @XmlRootElement(name = "instance")
 public class InstanceInfo {
 
+    /** @since 11.5 */
+    protected static final String DISTRIBUTION_PROPS = "distribution.properties";
```

Comment:
Why not defining it in ConfigurationConstants too?

Reply:
Indeed 🙂.